geomstats/geometry/general_linear.py

```diff
@@ -7,7 +7,7 @@
 class GeneralLinear(Matrices):
     """Class for the general linear group GL(n)."""
 
-    def __init__(self, n):
+    def __init__(self, n, **kwargs):
```

Comment:
If this was to get rid of the DeepSource issue, this is not needed anymore.

Reply:
no this is for initializing all parent classes with one call of super